README.md

```diff
@@ -1,34 +1,60 @@
-# Laserfiche Repository Client API Sample - Service App
-Sample project for demonstrating how to work with our Repository API client.
+# lf-sample-repository-api-dotnet-srv
 
-## Steps to recreate this projecct
+Sample .NET core service app that connects to a Laserfiche Cloud Repository using a service principal account.
+[Sample Code](./Program.cs)
 
-### Scafolding
+### Prerequisites
 
-```
-dotnet new console --framework net6.0
-```
+#### Software Prerequisites
 
-### Run the project
+- Visual Studio Code
+- .NET 6.0 core SDK
+- Git
+- Clone this repo on your local machine
 
-```
-dotnet run
-```
+#### 1. Create a Service Principal
 
-### Add dependencies
+- Login to account using web client as an administrator
```

Comment:
Since earlier we used "Laserfiche Repository Client", I guess since Web Client is also a product, we should use title case here. So, "Web Client".